src/style.css

```diff
@@ -0,0 +1,46 @@
+body {
+  margin: 0;
+}
+
+.star {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+}
+
+.stars--1 div:nth-child(1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 div:nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 div:nth-child(-n+3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 div:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 div:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
```
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
  background: url(images/star-active.svg);
}
```
use class selectors instead of `div` and unite repeated code